src/Testcontainers.Weaviate/WeaviateBuilder.cs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+using Docker.DotNet.Models;
+using DotNet.Testcontainers.Builders;
+using DotNet.Testcontainers.Configurations;
```

Comment:
The convention in the repo seems to favour all imports to be in a common `Usings.cs` file per module. please see an example [here](https://github.com/testcontainers/testcontainers-dotnet/blob/develop/src/Testcontainers.PostgreSql/Usings.cs)